lib/trino-filesystem-manager/src/main/java/io/trino/filesystem/manager/FileSystemModule.java

```diff
@@ -89,6 +91,11 @@ protected void setup(Binder binder)
 
         var factories = newMapBinder(binder, String.class, TrinoFileSystemFactory.class);
 
+        if (config.isAlluxioEnabled()) {
```

Comment:
Currently we've allowed this filesystem implementation to be additionally wrapped by the local filesystem cache implementation as well.
Are there benefits to allowing Alluxio filesystem and local filesystem cache to be configured together or should we disallow local filesystem cache when Alluxio filesystem is configured ?

Reply:
I think it would be OK to allow user to enable the two configurations together, as Alluxio FileSystem provides a Cluster-Level caching, while local filesystem cache provides a local cache. So we provide two different levels of caching in this way.